src/Integration.Vsix.UnitTests/CFamily/CLangAnalyzerTests.cs

```diff
@@ -37,27 +37,15 @@ namespace SonarLint.VisualStudio.Integration.Vsix.CFamily.UnitTests
     [TestClass]
     public class CLangAnalyzerTests
     {
-        private Mock<ITelemetryManager> telemetryManagerMock;
-        private TestLogger testLogger;
-        private Mock<IAnalysisStatusNotifier> analysisNotifierMock;
-        private Mock<ICFamilyIssueToAnalysisIssueConverter> cFamilyIssueConverterMock;
-        private Mock<IRequestFactoryAggregate> requestFactoryAggregateMock;
-
-        [TestInitialize]
-        public void TestInitialize()
```

Comment:
I changed the tests to remove the `TestInitialize`, and instead use the `CreateTestSubject(ISomething something = null ...)` pattern.